screens/Menu/AboutUs.js

```diff
@@ -35,23 +37,28 @@ function AboutUs (props) {
             "Hussain Witwit - hussainwitwit"
         ]
     }];
+    const textParagraph = [{
```

Comment:
Why is this not simply a string?

Reply:
Since I'm using a sectionList we need to pass it an array, not just a string